app/src/androidTest/java/edu/nextstep/camp/calculator/MainActivityTest.kt

```diff
@@ -1,130 +1,53 @@
 package edu.nextstep.camp.calculator
 
-import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
 import androidx.test.espresso.action.ViewActions
-import androidx.test.espresso.assertion.ViewAssertions
-import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
 import androidx.test.ext.junit.rules.ActivityScenarioRule
-import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.junit.runners.Parameterized.Parameters
+
+@RunWith(Parameterized::class)
+class MainActivityTest(input: Pair<Int, String>) {
+    private var buttonId: Int
+    private var experted: String
```

Comment:
```suggestion
class MainActivityTest(
    private val buttonId: Int
    private val experted: String
)
```
생성자를 위와 같이 구성하는 것이 더 직관 적으로 보입니다 :)